### PR TITLE
Fix #176423 leingpt.ru/chat

### LINKED ISSUE #176423: leingpt.ru
### Issue URL (Anti Adblock Script)
 [https://leingpt.ru/chat/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fleingpt.ru%2Fchat%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for Windows v7.17 Beta 3 (4657)
Browser: | Яндекс Браузер
License type: | paid
AdGuard driver: | WFP
Stealth mode options: | Block WebRTC, <br>Block Push API, <br>Block Location API, <br>Block Flash, <br>Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Hide your Referrer from third-parties (leingpt.ru), <br>Hide your User-Agent, <br>Hide your IP address (208.54.104.184), <br>Remove X-Client-Data header from HTTP requests, <br>Protect against DPI, <br>Self-destructing third-party cookies (180), <br>Block trackers, <br>Disable WAP Push Message Routing Service, <br>Disable Windows Defender automatic samples submission, <br>Disable Windows Telemetry, <br>Turn off Advertising ID
DNS filtering: | server: `System`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Language-specific:</b><br/>AdGuard Russian
Userscripts: | https://userscripts.adtidy.org/beta/assistant/4.3/assistant.user.js<br>https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js<br>https://userscripts.adtidy.org/beta/popup-blocker/2.5/popupblocker.user.js
Other extensions used: | Касперский, adguard windows

### PR DESCRIPTION
**Pull request by SpotlightForBugs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [x] AdGuard gets detected on a website;
- [x] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://github.com/AdguardTeam/AdguardFilters/issues/176423

### Add your comment and screenshots

- Fixed AdGuard detected by removing the id `reklama` from a div
- then removed the ads by adding `https://yandex.ru/ads/system/context.js` to the `CyrillicFilters/common-sections/general_url.txt`

Screenshots

<details>

<summary>Screenshot 1 (AdGuard enabled --> Adblock detected:</summary>

image

</details>

<details>

<summary>Screenshot 2 (Adblocking fails):</summary>

image

</details>

<details>

<summary>Screenshot 3 (My fix):</summary>

image

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: CyrillicFilters/common-sections/general_url.txt [hunk 1]
```diff
@@ -785,6 +785,7 @@ _box_ads/
 _sponsored_links.
 ban_click
 ucoz.ru/informer/
+||yandex.ru/ads/system/context.js$third-party
```
Comment (Alex-302) [inline CyrillicFilters/common-sections/general_url.txt:788]: Избыточное. В базовом есть нужное правило.

File: CyrillicFilters/RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -24,6 +24,7 @@ happycoin.club##a[href][target="_blank"][rel="noopener"] > img[width][height]
 happycoin.club#?#.theiaStickySidebar > div.widget_text:has(> div.widget-title-cover > h5:contains(Реклама))
 novate.ru#?##right > h2:contains(Рекомендуем)
 ||myfanstv.net/uploads/*/baner/
+leingpt.ru#%#//scriptlet('remove-attr', 'id', 'div[id="reklama"]')
```
Comment (Alex-302) [inline CyrillicFilters/RussianFilter/sections/specific.txt:27]: `leingpt.ru#@##reklama` достаточно. ID не добавляется скриптом - в скриптлете особого смысла нет, зато не на всех платформах сработает.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: CyrillicFilters/RussianFilter/sections/specific.txt (hunk 2 above)
Comment (SpotlightForBugs) [reply CyrillicFilters/RussianFilter/sections/specific.txt:27]: You'd suggest using just leingpt.ru#@##reklama as the rule? I can't understand Russian, I am sorry

“Вы предложили бы использовать только leingpt.ru#@##reklama как правило? Простите, я не понимаю по-русски.”

**Comment by SpotlightForBugs** [comment]
Done

File: CyrillicFilters/RussianFilter/sections/specific.txt [hunk 3]
```diff
@@ -24,6 +24,7 @@ happycoin.club##a[href][target="_blank"][rel="noopener"] > img[width][height]
 happycoin.club#?#.theiaStickySidebar > div.widget_text:has(> div.widget-title-cover > h5:contains(Реклама))
 novate.ru#?##right > h2:contains(Рекомендуем)
 ||myfanstv.net/uploads/*/baner/
+leingpt.ru#@##reklama
```
Comment (Alex-302) [inline CyrillicFilters/RussianFilter/sections/specific.txt:27]: We add anti-adblock rules here (comment with issue link + rule/s):
https://github.com/AdguardTeam/AdguardFilters/blob/5136e7865d95a5882c81da8aa35795bea7e236f8/CyrillicFilters/RussianFilter/sections/antiadblock.txt#L28

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**